src/librustc_resolve/build_reduced_graph.rs

```diff
@@ -628,19 +629,21 @@ impl<'a> Resolver<'a> {
                                          expansion,
                                          block.span);
             self.block_map.insert(block.id, module);
-            self.current_module = module; // Descend into the block.
+            // Descend into the block.
+            self.current_module = module;
         }
     }
 
     /// Builds the reduced graph for a single item in an external crate.
     fn build_reduced_graph_for_external_crate_def(&mut self, parent: Module<'a>, child: Export) {
         let Export { ident, def, vis, span } = child;
-        // FIXME: We shouldn't create the gensym here, it should come from metadata,
+        // FIXME: we shouldn't create the gensym here, it should come from metadata,
         // but metadata cannot encode gensyms currently, so we create it here.
         // This is only a guess, two equivalent idents may incorrectly get different gensyms here.
         let ident = ident.gensym_if_underscore();
         let def_id = def.def_id();
-        let expansion = Mark::root(); // FIXME(jseyfried) intercrate hygiene
+        // FIXME(jseyfried): inter-crate hygiene.
```

Comment:
Please don't move trailing comments into separate lines.

Reply:
Well, trailing comments are discouraged in all the codebases I have contributed to, I'll say that much. It decreases readability and is especially problematic when there's a line length cap, I find. It may be worth a brief debate.